package.json

```diff
@@ -16,6 +16,9 @@
     "url": "https://github.com/testdouble/teenytest-promise/issues"
   },
   "homepage": "https://github.com/testdouble/teenytest-promise#readme",
+  "peerDependencies": {    
+    "teenytest": "^5.0.0"
```

Comment:
Agree! The peerdeps give us a way to say which version range of teenytest this plugin is compatible with.